crates/parry2d/Cargo.toml

```diff
@@ -68,4 +85,95 @@ simba = { version = "0.9", default-features = false }
 oorandom = "11"
 ptree = "0.4.0"
 rand = { version = "0.8" }
-macroquad = "0.4"
\ No newline at end of file
+macroquad = "0.4"
+
+[package.metadata.docs.rs]
+rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+all-features = true
+cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
```

Comment:
You don’t need to pass a cfg, docs.rs does that already.

I don’t think you can use all-features due to dim2/dim3 and f32/f64

you can do this like I do elsewhere:

```

[package.metadata.docs.rs]
rustdoc-args = [ "-Zunstable-options", "--generate-link-to-definition"]
cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
```